docs/pages/concept/web.md

```diff
@@ -0,0 +1,145 @@
+
+Before you start using web renderer, it's crucial to ensure that you have a compositor version with a built-in web rendering feature. Due to web renderer being complex and large in size we offer 2 version of the compositor, one without web renderer and another one with web renderer.
+
+While [web renderer](../api/renderers/web.md) renders websites, the [web view](../api/components/WebView.md) component presents the rendered website and is used in the scene definition.
+
+:::tip
+You can view working example [here](https://github.com/membraneframework/video_compositor/blob/master/examples/web_view.rs)
+:::
+
+## Using web renderer & web view
+Firstly, the web renderer instance has to be registered:
+```typescript
+{
+    "type": "register",
+    "entity_type": "web_renderer",
+    "instance_id": "example_website",
+    "url": "https://example.com",
+    "resolution": {
+        "width": 1920,
+        "height": 1080
+    }
+}
+```
+- `instance_id` - unique renderer identifier, which is used to refer to the web renderer.
+- `url` - website url. All protocols supported by chromium can be used
+
+Now, we can define scene with web view component:
+```typescript
+{
+    "type": "update_scene",
+    "outputs": [
+        {
+            "output_id": "output_1",
+            "root": {
+                "id": "embed_input_on_website",
+                "type": "web_view",
+                "instance_id": "example_website",
+            }
+        }
+    ]
+}
+```
+
+`instance_id` - the ID of previously registered web renderer.
+
+That's it!
+
+:::warning
+One specific web renderer instance can be used only by one component!
+:::
+
+
+## Embedding other components
+Embedding is a process of rendering children components on a website.
+
+There are 3 embedding methods available:
+- `chromium_embedding` - the most flexible method. The children are passed directly to the chromium instance and rendered on HTML canvas. You can animate them and use CSS styles without any problems. However, while it offers the best flexibility, it's also the slowest method.
+- `native_embedding_over_content` - renders the website first and then renders child components on top of the website. It's faster than `chromium_embedding`, but CSS styling does not work.
+- `native_embedding_under_content` - renders component children first and then the website on top of them. The website needs a transparent background. Performance and limitations are the same as for `native_embedding_over_content`.
+
+The default embedding method is `native_embedding_over_content`.
+You can change the embedding method in the [register renderer request](../api/routes#register-renderer). For example:
+```typescript
+{
+    "type": "register",
+    "entity_type": "web_renderer",
+    "instance_id": "example_website",
+    "url": "https://example.com",
+    "resolution": {
+        "width": 1920,
+        "height": 1080
+    },
+    // NEW
```

Comment:
is there sth docusaurs specific that would highlight it better?

Reply:
I could use line highlights in code blocks
<img width="709" alt="Screenshot 2024-01-11 at 4 38 59 PM" src="https://github.com/membraneframework/video_compositor/assets/16838075/fafba337-d608-49d2-a4cc-5075c845b13e">